templates/definition/meter/quatt.yaml

```
@@ -0,0 +1,43 @@
+template: quatt
+products:
+  - brand: Quatt
+    description:
+      generic: Hybrid Heatpump
+group: meter
+requirements:
+  evcc: ["skiptest"]
+  description:
+    en: |
+      Integration for Quatt hybrid heat pumps via local API.
+      This read-only meter reports the heat pump's current power input and water outlet temperature.
+      No authentication required.
+    de: |
+      Integration für Quatt-Hybrid-Wärmepumpen über die lokale API.
+      Dieses read-only Meter liefert die aktuelle Leistungsaufnahme und die Vorlauftemperatur der Wärmepumpe.
+      Keine Authentifizierung erforderlich.
+params:
+  - name: host
+    required: true
+    default: REQUIRED
+    description:
+      en: IP or hostname
+      de: IP-Adresse oder Hostname
+    help:
+      en: IP address or hostname of the Quatt controller in your local network.
+      de: IP-Adresse oder Hostname des Quatt-Controllers im lokalen Netzwerk.
+  - name: port
+    required: false
+    default: 8080
+    description:
+      en: API port
+      de: API-Port
+    help:
+      en: Port used by the local Quatt API (default is 8080).
+      de: Port der lokalen Quatt-API (Standard ist 8080).
```

```suggestion
  - name: port
    default: 8080
```
Can use the default description for host and port here. The help texts should be obvious because there are no other url at play here.